src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.html

```diff
@@ -11,10 +11,12 @@
     
     <c-ge-form-renderer
             batch-id={recordId}
-            onsubmit={handleSubmit}
+            onbatchsubmit={handleBatchSubmit}
+            onsinglesubmit={handleSingleSubmit}
```

Comment:
I don't know if it is doable in this PR, but I'd love to see this be a single handler eventually for `onsubmit` since the App should already know whether it is in batch or single mode and what to do next in each case.

Reply:
Rolled this back into a single `onsubmit`. Checking the `isBatchMode` getter within the submit handler to know which path to go down.